### PR TITLE
adding a tip about powertoys calculator plugin

### PR DESCRIPTION
**Pull request by jjavierdguezas**
- related to [issue microsoft#20187](https://github.com/microsoft/PowerToys/issues/20187) and [PR #24655](https://github.com/microsoft/PowerToys/pull/24655)

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
@mattwojo - Could you review this PR?

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by mattwojo** [comment]
Hold off on merging until PT v0.69 is released, which will include this functionality.

**Comment by Jak-MS** [comment]
@mattwojo - reapplying the triaged label to remove this from our queue.  we'll see it again when you sign off.

File: hub/powertoys/run.md [hunk 1]
```diff
@@ -166,6 +166,9 @@ If the program plugin's option "Include in global result" is not selected, inclu
 > Please be aware of the different decimal and thousand delimiters in different locals.
 > If your system's number format uses the comma (`,`) as the decimal delimiter, you have to write a space between the number(s) and comma(s) on operations with multiple parameters. The input has to look like this: `min( 1,2 , 3 , 5,7)` or `min( 1.2 , 3 , 5.7)`.
 
+> [!TIP]
+> The Calculator plugin can handle some human multiplication expressions like `2(3+4)` and `(1+2)(3+4)` by inserting the `*` operator where appropriate.
```
Comment (Jay-o-Way) [inline hub/powertoys/run.md]:
```suggestion
> The Calculator plugin can handle some implied multiplication expressions like `2(3+4)` and `(1+2)(3+4)` by inserting the multiplication operator where appropriate.
```
Just a suggestion

**Comment by crutkas** [comment]
0.69 is released,

**Review by mattwojo — APPROVED** [review]

**Pull request merged**